tutorial_plugin_java.md

```diff
@@ -0,0 +1,160 @@
+﻿---
+layout: page
+title: Tutorial Plugin Java
+permalink: /plugin/tutorial/java
+---
+
+# How to create a plugin for SmartSHARK
+This tutorial explains how to create a Java plugin for SmartSHARK. This short readme is a tutorial on how to program a plugin starting from the ground. The first section will provide you with the basic information. More advanced information will be outlined for each file in the files section.
```

Comment:
Second sentence: "We explain how you can create a simple plugin for SmartSHARK using the minimal working example [jmweSHARK](LINK HERE).